README.md

```diff
@@ -1,9 +1,33 @@
 # Budget-app
-you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+I've been working on this RoR Capstone for the last 3 days, the app is about a Bugdet-app.
+
+There is a splash home page where you have the option to log_in and sign_up in the app. If you don't have an account you can't visit the webpage.
+
+The sig_up option has a email-validation, were I added the gem `letter-opener` to acept the email confirmation.
+
+When a user is Log_in you can visit the app and work in yours categories and transactions.
+
+Also this have a database on Postresql, where the data is stored an avaible to use.
+
+You can create a category and add expenses to it.
+
+You can see the total amount of your expenses of each category, and the total amount of all the expenses in general.
+
+The style followed is a mobile-first structure.
+
+## Pesentation
+
+https://www.loom.com/share/1492d3e6e83e40bcaa6131f4c9410839
 
 ## Live Version
 
-[Avaible Soon]()
+[Avaible Soon on Heroku]()
```

Comment:
- [ ] Please provide us with a proper link to your application. This is one of the basic requirements for approval of this project.

Reply:
Done! https://bregorn-budget-app.herokuapp.com/